script.js

```diff
@@ -1,4 +1,66 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // 1 = Scissors, 2 = Paper, 3 = Stone
+  var randomAction = randomSPS();
+  myOutputValue =
+    'Error. Please input only "Scissors", "Paper" or "Stone". Try agin!';
+  if (randomAction == 1) {
+    actionThrown = "Scissors";
+  }
+  if (randomAction == 2) {
+    actionThrown = "Paper";
+  }
+  if (randomAction == 3) {
+    actionThrown = "Stone";
+  }
+  console.log("action: " + actionThrown);
+  // when user wins
+  if (
+    (input == "Scissors" && actionThrown == "Paper") ||
+    (input == "Stone" && actionThrown == "Scissors") ||
+    (input == "Paper" && actionThrown == "Stone")
+  ) {
+    var myOutputValue = "User win";
+  }
+  // when program wins
+  if (
+    (input == "Scissors" && actionThrown == "Stone") ||
+    (input == "Paper" && actionThrown == "Scissors") ||
+    (input == "Stone" && actionThrown == "Paper")
+  ) {
+    var myOutputValue = "Program Win";
+  }
+  if (input == actionThrown) {
+    // draw
+    var myOutputValue = "It is a draw";
```

Comment:
you don't have to use var again once you have declared it :)

Let me know if this makes sense!